tests/backup/backup_restore_basic_test.go

```diff
@@ -1010,33 +1048,36 @@ var _ = Describe("{AllNSBackupWithIncludeNewNSOption}", func() {
 			scheduleName = fmt.Sprintf("%s-schedule-%v", BackupNamePrefix, time.Now().Unix())
 			namespaces := []string{"*"}
 			labelSelectors := make(map[string]string)
-			err = CreateScheduleBackup(scheduleName, appClusterName, backupLocationName, backupLocationUID, namespaces,
-				labelSelectors, orgID, "", "", "", "", periodicSchedulePolicyName, periodicSchedulePolicyUid, ctx)
+			err = CreateScheduleBackup(scheduleName, destinationClusterName, backupLocationName, backupLocationUID, namespaces,
+				labelSelectors, orgID, "", "", "", "", schedulePolicyName, schedulePolicyUid, ctx)
 			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of schedule backup with schedule name [%s]", scheduleName))
 			firstScheduleBackupName, err := GetFirstScheduleBackupName(ctx, scheduleName, orgID)
 			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the name of the first schedule backup [%s]", firstScheduleBackupName))
 			scheduleNames = append(scheduleNames, scheduleName)
 		})
 		// To ensure applications are deployed after a schedule backup is created
-		Step("Schedule applications to create new namespaces", func() {
-			log.InfoD("Scheduling applications to create new namespaces")
-			contexts = make([]*scheduler.Context, 0)
-			for i := 0; i < Inst().GlobalScaleFactor; i++ {
-				taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+		Step("Schedule applications in destination cluster to create new namespaces", func() {
+			log.InfoD("Scheduling applications in destination cluster to create new namespaces")
+			err := SetDestinationKubeConfig()
```

Comment:
Same here

Reply:
Reduced context switches. I think it is better to add a context switch where needed. It is uncertain whether some function switches the context before reaching the step, so we need to confirm that we are using the intended context.